adventure-crew-game/Assets/Scripts/Backend/IAP/IAPProductConfig.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Linq;
+using UnityEngine;
+
+namespace Backend.IAP
+{
+    [CreateAssetMenu(fileName = "IAP Product Config", menuName = "IAP/Product Config", order = 0)]
+    public class IAPProductConfig : ScriptableObject
+    {
+        [SerializeField] private IAPProduct[] products;
+
+        public IAPProduct[] Products => products;
+
+        internal IAPProduct GetProductById(string id)
+        {
+            return products.FirstOrDefault(product => product.Id == id);
+        }
+
+        internal bool IsProductPurchased(string id)
+        {
+            var product = GetProductById(id);
+            return product?.Purchased ?? false;
+        }
+
+        [ContextMenu("To JSON")]
+        public void ToJson()
```

Comment:
Can't see any references to this function. What are your plans for this function?

Reply:
Oh, was trying something to save the data about the IAP products being purchased or not.
Eventually went with PlayerPrefs.
I'll remove it

Reply:
Removed it